README.md

```diff
@@ -0,0 +1,125 @@
+- **Count Cells in Dataset**: Count the number of cells present by combining the enhanced dataset with binary maps. The plugin enables users to focus on regions of interest and generate accurate cell counts.
+
+
+## User's manual
+
+### Installation
+
+Install [napari](https://napari.org/dev/tutorials/fundamentals/installation.html)  and then install this plugin using the following command
+
+
+### How to use
+
+1. Run napari from your python environment
+2. Select the Plugin option located at the top menu bar
+3. Select Jaworski lab cell plugin
+
+   <img src="images/README/napari_img.png" alt="Image description" width="500" height="300">
+4. After a few seconds the widgets will be added at the right side of the napari window
+   
+   <img src="images/README/plugin_img.png" alt="Image description" width="200" height="300">
+
+5. The following image illustrates the series of steps to segment, label, and count cells in the dataset.
+   <img src="images/README/pipeline.png" alt="Image description" width="630" height="200">
+
+### Load dataset
+
+The top widget includes a button to open a file selection window. Click on "Select File," choose the .oir dataset, and then click the "Open BioImage" button in the widget.
+   
+   <img src="images/README/load_dataset_widget.png" alt="Image description" 
+   width="280" height="160">
+
+
+### Pre Process data
+
+This widget allows you to modify various values to adjust the contrast of the dataset and apply a Gaussian filter. By default, we have set values that work best for the Jaworski lab datasets. You don’t need to modify these values, but feel free to adjust them if needed. Once ready, press the "Pre-process BioImage" button, and additional layers will be added to the visualizer.
+
+ <img src="images/README/preprocess_widget.png" alt="Image description" 
+   width="280" height="160">
+
+ <img src="images/README/preprocess_result.png" alt="Image description" 
+   width="480" height="300">
+
+This process creates 2 additional layers (Keep track of these):
+- Pre-processed Data: An enhanced version of the dataset that sharpens cell borders and brightens the interior of blobs to help identify multiple cells within a single blob.
+- Cloud Binary: A binary image representing the region of interest with the highest probability of containing cells.
+
+
+### Data Segmentation
+
+For this step we use [napari-CellSeg3D](https://www.napari-hub.org/plugins/napari-cellseg3d) plugin, which is automatically loaded. Please refer to their documentation for guidance on how to use it and achieve optimal segmentation results. By default, this plugin loads the best values for the Jaworski Lab datasets. Feel free to modify these values if necessary. Click the "Start" button and wait for the process to complete (note that it may take considerable time if you are not using a GPU with CUDA support) before proceeding to the next step.
```

Comment:
```suggestion
The [napari-CellSeg3D](https://www.napari-hub.org/plugins/napari-cellseg3d) plugin is used for segmentation and is automatically loaded. Please refer to their documentation for guidance on usage and achieving optimal segmentation results. By default, this plugin loads the best values for the Jaworski Lab datasets. Click the "Start" button and wait for the process to complete. It may take a long time to finish if you are not using a GPU with CUDA support.
```